daily-challenges/2024-11/04/ducnq.k11.spec.ts

```
@@ -0,0 +1,35 @@
+
+function calculateMagicGate(array: string[]): number[]{
+    let results: number[] = [];
+    for(let i of array){
+        i = i.replace(/\s/g, "");
+        let calculate = new Function(`return (${i})`);
+        let result = calculate().toFixed(2);
+        results.push(result);
+    }
+    return results;
+}
+console.log(calculateMagicGate(["10/3", " 5*5 ", "100 / 4"])); 
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'calculate' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)